core/computed/trace-engine-result.js

```diff
@@ -47,6 +49,16 @@ class TraceEngineResult {
         },
       },
       lanternSettings,
+      async resolveSourceMap(params) {
+        const sourceMap = SourceMaps.find(sm => sm.scriptId === params.scriptId);
+        if (!sourceMap) {
+          return null;
+        }
+
+        const compiledUrl = sourceMap.scriptUrl || 'compiled.js';
+        const mapUrl = sourceMap.sourceMapUrl || 'compiled.js.map';
```

Comment:
Seems kinda weird to fallback to `compiled.js` as the default script URL. Would this happen for inline scripts and stuff?